packages/loader/container-loader/src/container.ts

```diff
@@ -814,6 +814,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             clientId: this.clientId,
         };
 
+        this.mc.logger.send(
+            {
+                eventName: "CloseAndGetPendingLocalState",
+                category: reason === undefined ? "generic" : reason,
```

Comment:
Do we have any idea what kinds of values will be used for "reason"? I'm not sure if category makes sense.

Reply:
disconnectFromDeltaStream in connectionManager has a similar behaviour having just a string to add some context.

Reply:
We should not use ITelemetryBaseLogger method, instead we should use appropriate ITelemetryLogger methods, which ensures that category can be one of 3 values. Putting anything else will ensure data will go to nowhere (at least in Bohemia case), as they created for us 3 tables to match existing categories.

I think we should revisit `category` being a string and instead use TelemetryEventCategory for it to avoid cases like that